procedures/release-duty/merge-duty/merge_duty.rst

```diff
@@ -27,8 +27,6 @@ nearly three weeks, with *three* major days of activity:
    -  `Do migration no-op trial runs <#do-migration-no-op-trial-runs>`__
    -  `Sanity check no blocking migration
       bugs <#sanity-check-no-blocking-migration-bugs>`__
-   -  `Land whatsnewpage list of
```

Comment:
Great to see this going away. If we ever need them, we can revert them from the git history.